rsc/src/main/java/com/cloudera/livy/rsc/RSCConf.java

```diff
@@ -51,8 +51,10 @@
 
     // Address for the RSC driver to connect back with it's connection info.
     LAUNCHER_ADDRESS("launcher.address", null),
+    LAUNCHER_PORT_RANGE("launcher.port.range", "10000~10110"),
+    // Setting up of this propety by user has no benefit. It is currently being used
+    // to pass  port information from ContextLauncher to RSCDriver
     LAUNCHER_PORT("launcher.port", -1),
```

Comment:
@zjffdu 
As mentioned in the Comment for launcher port . This property is still being used to pass   port information from contextlauncher to RSCDriver. 
ContextLauncher.java ,set this property 
conf.set(LAUNCHER_PORT, factory.getServer().getPort());

And RSCDriver takes that information 
int launcherPort = livyConf.getInt(LAUNCHER_PORT);

So this property still needed. 
But user setting it up will have no benefit. So we should not mention this property in livy-client.conf.template.

Please let me if its ok .

Other comments are indentation related , I'll fix them and push .